deltachat-ios/Controller/ChatListController.swift

```diff
@@ -157,8 +161,9 @@ extension ChatListController: UITableViewDataSource, UITableViewDelegate {
         }
 
         let chatId = chatList.getChatId(index: row)
+        let chat = DcChat(id: chatId)
         if chatId == DC_CHAT_ID_ARCHIVED_LINK {
-            return getArchiveCell(tableView)
+            return getArchiveCell(tableView, title: chat.name)
```

Comment:
wow, the chat name has now the translated string for "Archived chats (#noOfChats)"?

Reply:
yip, exactly :)